packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,58 @@ class FancyFocus extends FocusManager {
+| direction     | 'column'\|'row'                                         | undefined | The navigation direction for focus ('left/right' or 'up/down') |
+| selectedIndex | number                                                  | undefined | The index of the currently selected item.                      |
+| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | undefined | Child element or elements of the FocusManager.                 |
+| wrapSelected  | boolean                                                 | false     | Enables wrapping behavior for focus navigation.                |
 
 ### Methods
 
-#### appendItems
+#### appendItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>): void
 
-Appends items to child list and updates focus
+This method appends the provided rows to the end of the FocusManager's children
+
+#### appendItemsAt(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>, idx: number): void
+
+This method is similar to appendItems, but it allows appending items at a specific index in the list of child items.
 
 ##### Arguments
 
-| name  | type                                                    | required | default | description     |
-| ----- | ------------------------------------------------------- | -------- | ------- | --------------- |
-| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | items to append |
+| name  | type                                                    | required | default   | description                   |
+| ----- | ------------------------------------------------------- | -------- | --------- | ----------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []        | The list of items to append   |
+| idx   | number                                                  | true     | undefined | The index to insert the items |
+
+#### prependItems(items: <DocsLink id="lng.Component">lng.Component[]</DocsLink>): void
+
+This method prepends items to the beginning of the FocusManager's children.
+
+| name  | type                                                    | required | default | description                 |
+| ----- | ------------------------------------------------------- | -------- | ------- | --------------------------- |
+| items | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | []      | The list of items to append |
+
+#### removeItemAt(index: number): void
+
+Removes an item at a specific index from the FocusManager's children.
+
+#### getTransitionXTargetValue(): string
+
+Returns the transition value for the x coordinate of the FocusManager's items.
+
+#### getTransitionYTargetValue(): string
+
+Returns the transition value for the y coordinate of the FocusManager's items.
 
-#### render
+#### \_render(): void
```

Comment:
Thanks for the update! Could the slash in the beginning get removed?

Reply:
Linter automatically does that when I save the file, I think I should enable it and push the change

Reply:
Ah I see seems like the slash isn't showing up in the actual documentation though so if the linter continues to add it I think it's fine